compiler/crew/README.md

```diff
@@ -0,0 +1,13 @@
+# crew
+
+_crew_ is circle partitioning Configuration REader and Writer library.
```

Comment:
```suggestion
_crew_ is circle partitioning Configuration Reader and Writer library.
```

Reply:
`RE` is intentionally set capital for `cREw`